lib/domain/core/entities/common_behavior_object.dart

```diff
@@ -1762,3 +1763,16 @@ class UserFeedStore {
 
   static final UserFeedStore _singleton = UserFeedStore._internal();
 }
+
+class AppBrand {
+  factory AppBrand() {
+    return _singleton;
+  }
+
+  AppBrand._internal();
+
+  BehaviorSubject<String> appLogo = BehaviorSubject<String>.seeded('');
```

Comment:
these values can be initialized to actual values instead of empty strings because they can be set/changed later